codalab/lib/worksheet_util.py

```diff
@@ -1054,12 +1054,12 @@ def flush_worksheets():
                         )
                         .data
                     )
-
+                    appended_directive_blocks_index = True
                     raw_to_block.append((len(blocks) - 1, 0))
                 else:
                     raise UsageError("unknown directive `%s`" % command)
                 # Add a placeholder for other cmds
```

Comment:
Sorry, I don't recall -- what's the point of this `None` placeholder again? Can you expand this comment so it's clear to future readers of the code what it's used for?

Reply:
Comment updated. Previously, when the item type is `directive`, `None` will always be added to `raw_to_block`, which is buggy since, for `addschema` / `search` cmd, some substantial items will be added to `raw_items`, so we need to update `raw_to_block` as well.